src/js/components/ModalComponent.jsx

```diff
@@ -51,18 +52,19 @@ var ModalComponent = React.createClass({
   },
 
   render: function () {
-    var modalDialogClassName =
-      "modal-dialog " + modalSizeClassName(this.props.size);
+    var props = this.props;
+    var isInState = this.state.isIn;
```

Comment:
Naming is a bit cryptic. `in` and `isIn` (and especially `isInState`) take too many braincycles to decode imho :stuck_out_tongue: 
How about `isShown` and `.shown`?